NuGetPackages/readme/MonoGame.Framework/readme.txt

```diff
@@ -7,18 +7,25 @@ The goal is to make it easy for XNA developers to create cross-platform games wi
 This package is just the Binaries for MonoGame for easier platform dll updates.  
 
 _Supported Platforms
-	* Windows GL
+Each NuGet now targets an individual platform to make for easier management and targeting.  It also allows for a more varied target group and some of the specialised platforms.
+
+The currentlt available packages are:
+	* Windows GL (OpenGL)
+	* Windows DX (DirectX)
 	* Windows 8 & 8.1
-	* Windows Phone 8, 8.1 and universal projects
+	* Windows Phone 8, 8.1 
+	* Windows Universal projects
 	* iOS
-	* MacOS
+	* MacOS (net4 / net45 / MonoMac and Xamarin.Mac)
 	* Android
+	* Ouya
+	* Linux
 
 _Contents
 This package contains only MonoGame DLL references for a solution.
-For project templates either use the Monogame Project templates or the Main MonoGame NuGet package
+For project templates either use the Monogame Project templates from the MSI installer on windows or the MonoDevelop/Xamarin Studio Addin on Mac
```

Comment:
There is no MSI installer that I know of... do you mean the MonoGameSetup.exe ?  Would just say "from the Windows installer".